projects/tinkerbell/pbnj/docker/linux/Dockerfile

```diff
@@ -0,0 +1,51 @@
+ARG BASE_IMAGE # https://gallery.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-base
+ARG BUILDER_IMAGE
+
+FROM $BUILDER_IMAGE as ipmitool-builder
+
+ARG IPMITOOL_CLONE_URL=https://github.com/ipmitool/ipmitool.git
+ARG IPMITOOL_GIT_TAG
+
+RUN yum install -y \
+    autoconf \
+    automake \
+    libtool \
+    ncurses-devel \
+    openssl-devel \
+    readline-devel \
+    readline
+
+RUN git clone $IPMITOOL_CLONE_URL && \
+    cd ipmitool && \
+    git checkout $IPMITOOL_COMMIT && \
+    ./bootstrap && \
+    ./configure \
+        --prefix=/usr/local \
+        --enable-ipmievd \
+        --enable-ipmishell \
+        --enable-intf-lan \
+        --enable-intf-lanplus \
+        --enable-intf-open && \
+    make && \
+    make install
+
+COPY --chmod=755 _output/dependencies/$TARGETOS-$TARGETARCH/github/grpc-ecosystem/grpc-health-probe/grpc_health_probe-linux-$TARGETARCH /usr/local/bin/grpc_health_probe
```

Comment:
Im not sure this is going to work. Unless this build is pulling in its deps statically, there may be some dynamic libs being pulled in.  run `ldd` against this and see.  Im not sure how i feel about this here, should we just make this its own project?

Reply:
I was thinkin the same too eventually. This PR is just an initial pass for kicking off a review, in no way final. For now, I just copied their upstream Dockerfile where they do a [wget](https://github.com/tinkerbell/pbnj/blob/main/Dockerfile#L49L51).